utils/event.ts

```diff
@@ -1,22 +1,77 @@
 export let event = {
-    httpMethod: '',
-    path: '',
-    queryStringParameters: {},
-    body: {},
-    requestContext: {},
+  httpMethod: "",
+  path: "",
+  queryStringParameters: {},
+  body: {},
+  requestContext: {},
 };
 
 export const setEvent = (rawEvent) => {
-    console.log(rawEvent);
+  console.log(rawEvent);
 
-    try {
-        rawEvent.body ? rawEvent.rawBody = rawEvent.body : null;
-        rawEvent.body ? rawEvent.body = JSON.parse(rawEvent.body) : null;
-        rawEvent.rawPath ? rawEvent.path = rawEvent.rawPath : null;
-        !rawEvent.httpMethod ? rawEvent.httpMethod = rawEvent.requestContext?.http?.method : null;
+  try {
+    rawEvent.body ? (rawEvent.rawBody = rawEvent.body) : null;
+    rawEvent.body ? (rawEvent.body = JSON.parse(rawEvent.body)) : null;
+    rawEvent.rawPath ? (rawEvent.path = rawEvent.rawPath) : null;
+    !rawEvent.httpMethod
+      ? (rawEvent.httpMethod = rawEvent.requestContext?.http?.method)
+      : null;
+  } catch {}
+
+  event = rawEvent;
+  return event;
+};
+
+export const extractPaginationParams = (event) => {
```

Comment:
Created this and the sortParams helper so we don't have to repeat the code for pulling the params in each route. Prioritizes body, falls back to query string, has safe defaults